scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -1641,7 +1641,10 @@ public void sendFlight(DTLSFlight flight) {
 			}
 
 			public void cancelRetransmissions() {
-				connection.cancelPendingFlight();
+				// TODO remove this check when this experimental feature will be
+				// not experimental anymore ^^
+				if (config.isEarlyStopRetransmission())
```

Comment:
Personally,  I prefer without brackets for "single line if", but if you prefer with it I can add it.